eslint.config.mjs

```diff
@@ -252,12 +252,10 @@ const config = createConfig([
       sourceType: 'module',
     },
   },
-  // Tailwind ESLint
+  // Tailwind ESLint for React Web
```

Comment:
Separating eslint tailwind plugin settings so we can point to each tailwind config separately